### PR TITLE
Notification when you are close to level

### PR DESCRIPTION
**Pull request by Vanillj**
It was a request here: [https://github.com/runelite/runelite/issues/14442](https://github.com/runelite/runelite/issues/14442)

This plugin will help users from missing when they reach a level and help prevent leveling by accident.

The plugin checks if the XP difference is less than some XP threshold the user can set (default to 5k). There is also a delay between the notification to prevent it from spamming each xp drop (defaults to 5 min).

### CODE REVIEW

File: plugins/Xp-To-Level-Notification [hunk 1]
```diff
@@ -0,0 +1,2 @@
+repository=https://github.com/Vanillj/XpToLevelNotification
```
Comment (Nightfirecat) [inline plugins/Xp-To-Level-Notification:1]: The repository URL is malformed
```suggestion
repository=https://github.com/Vanillj/XpToLevelNotification.git
```

**Comment by Nightfirecat** [comment]
Additionally, plugin file names must be lowercase alphanumeric + dashes. try: "xp-to-level-notification"

File: plugins/Xp-To-Level-Notification (hunk 1 above)
Comment (Vanillj) [reply plugins/Xp-To-Level-Notification:1]: I appreciate the correction.

**Comment by Vanillj** [comment]
Overlooked that, it should work now.

**Pull request merged**